src/core/generator.js

```diff
@@ -1,23 +1,34 @@
 import { buildRandomizer } from "./randomizer";
-import { messages } from "./messages";
+import { messages, emojis } from "./messages";
 /**
  * Setup generator function that takes some config and adds logic to generate message
  *
  * @param {object} config Configuration for generator
  * @param {string} config.generateButtonSelector Selector for generate new message button
+ * @param {string} config.generateEmojiButtonSelector Selector for generate new emoji button
  * @param {string} config.copyButtonSelector Selector for copy button
  * @param {string} config.resultContainer Selector for container where we put our result message
+ * @param {string} config.resultEmojiContainer Selector for container where we put our result message
  */
 function setupGenerator(config) {
   const generateButton = document.querySelector(config.generateButtonSelector);
+  const generateEmojiButton = document.querySelector(config.generateEmojiButtonSelector);
   const copyButton = document.querySelector(config.copyButtonSelector);
   const output = document.querySelector(config.resultContainer);
+  const emojiOutput = document.querySelector(config.resultEmojiContainer);
   const randomizer = buildRandomizer(messages);
-
+  let index = 0;
   function generateNewMessage() {
     copyButton.classList.remove("btn-copied");
     const { newMessage } = randomizer.getRandomMessage();
-    output.innerText = newMessage; // message
+    output.innerText = newMessage;
+  }
+  function generateNewEmoji() {
+    copyButton.classList.remove("btn-copied");
+    console.log(index);
+    emojiOutput.innerHTML = emojis[index];
+    if (index >= emojis.length - 1) index = 0;
```

Comment:
Instead of writing entire logic here, let move it to randomizer file and add new function `randomizer.getRandomEmoji()`. And I'd suggest to use `{ }` inside `if else` statement even if there is only one line of code. As it'll be more readable with more complex conditions